config/locales/fr.yml

```diff
@@ -9,6 +9,7 @@ fr:
     all: Tout
     analytics: Analytique
     browse: Parcourir
+    code_of_conduct: Code de conduite
```

Comment:
This translation seems not reflected on:
https://github.com/forem/forem/blob/43adc9f1fbe5667340dca39cc7dcd50741d7e486/config/locales/views/articles/fr.yml#L16